opendatahub-io/odh-dashboard
# Configure pipelines modal

closes: https://github.com/opendatahub-io/odh-dashboard/issues/1060
closes: https://github.com/opendatahub-io/odh-dashboard/issues/1062

## Description


Added modals for creating and removing a pipeline server

![Screenshot 2023-04-26 at 4 10 28 PM](https://user-images.githubusercontent.com/12587674/234860322-f29db1be-8cdb-4ef2-9221-64fedd29bc16.png)
![Screenshot 2023-04-26 at 4 13 31 PM](https://user-images.githubusercontent.com/12587674/234860563-a6b17072-ff98-4c41-8c57-d002c16db0f1.png)
![Screenshot 2023-04-26 at 4 13 55 PM](https://user-images.githubusercontent.com/12587674/234860584-2f94b602-28a7-4ece-b07e-81f3993439ad.png)

## How Has This Been Tested?



1. create a new project
2. go to `/pipelines`
3. _Configure pipeline server_
4. create new object storage
5. input **real** aws data connection info
6. configure
7. should configure (might start as an error but this is a bug in error handling)
8. go delete it from the dropdown in the right

## Test Impact


n/a for pipelines right now


## Request review criteria:




/cc @kywalker-rh @vconzola @andrewballantyne
